DotNet/mapping/types.js

```diff
@@ -41,7 +41,8 @@
         name: () => 'SessionResults',
     },
     "TestResults": {
-        name: () => 'TestResults',
+        name: () => `TestResults`,
+        get: (target, key) => key.startsWith('is') ? `${target}.${key}()` : simpleGetter(target, key)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9645502)
```suggestion
        get: (target, key) => key.startsWith("is") ? `${target}.${key}()` : simpleGetter(target, key)
```